api_practice/routers/health_check.py

```diff
@@ -0,0 +1,18 @@
+from fastapi import APIRouter, status
+
+from api_practice.schemas import HealthCheckStatus
+from api_practice.log import LoggerConfig
+
+router = APIRouter(
+    tags=['Health_Check'],
+    prefix='/social_api'
+)
+msg_log = LoggerConfig()
```

Comment:
wee must not initialize the logger class again here.  It must be started only in main class. 
In all other classes it must be just "logger = logging.getLogger(__name__)"
and then logger.info('xxxxxxx')

Reply:
Modified